test/contracts/OVM/verification/OVM_FraudVerifier.spec.ts

```diff
@@ -178,6 +178,54 @@ describe('OVM_FraudVerifier', () => {
           ).to.equal(Mock__OVM_StateTransitioner.address)
         })
       })
+
+      describe('when provided an invalid transaction proof index', () => {
+        before(() => {
+          Mock__OVM_CanonicalTransactionChain.smocked.verifyTransaction.will.return.with(
+            false
+          )
+        })
+
+        it('should revert', async () => {
+          let transactionDummyBatchProof = DUMMY_BATCH_PROOFS[0];
+          let preStateDummyBatchProof = DUMMY_BATCH_PROOFS[0];
+          transactionDummyBatchProof.index = 11;
+          preStateDummyBatchProof.index = 3;
+          await expect(
+            OVM_FraudVerifier.initializeFraudVerification(
+              NULL_BYTES32,
+              DUMMY_BATCH_HEADERS[0],
+              preStateDummyBatchProof,
+              DUMMY_OVM_TRANSACTIONS[0],
+              DUMMY_TX_CHAIN_ELEMENTS[0],
+              DUMMY_BATCH_HEADERS[0],
+              transactionDummyBatchProof
+            )
+          ).to.be.revertedWith('Invalid transaction inclusion proof.')
+        })
+      })
+    })
+
+    describe('when provided an invalid transaction', () => {
```

Comment:
I think these is a good test to add--let's get a little more specific though!  I think if we did two cases:
- `describe('when provided an invalid transaction proof')`
- `describe('when provided an invalid state root proof')`
and had `revertedWith` checks on each, that would be super dope.

Reply:
so I didn't see it before, but I think this is already done...
line 105 is 'invalid prestate root proof' and 'valid tx proof'
line 134 is 'valid prestate root roof' and 'invalid tx proof'